### PR TITLE
OCPBUGS-34684: i18n upload/download routine task - Sprint 254

### PR DESCRIPTION
**Pull request by cyril-ui-developer**

### CODE REVIEW

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by jhadvig** [comment]
```
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  cluster-settings/alertmanager/recei      01:55        1        -        1        -        - │
  │    vers/webhook.cy.ts                                                                          │
  └──────────────────────────────
```

File: frontend/public/locales/zh/public.json [hunk 1]
```diff
@@ -1238,7 +1243,6 @@
   "Exceptions": "例外",
   "with exceptions": "带有例外",
   "Any port": "任何端口",
-  "NetworkPolicy details": "网络策略详情",
```
Comment (XiyunZhao) [inline frontend/public/locales/zh/public.json:1241]: This translation is missing in 4.17 on NetworkPolicy details -> Details Tab

Comment (cyril-ui-developer) [reply frontend/public/locales/zh/public.json:1241]: > This translation is missing in 4.17 on NetworkPolicy details -> Details Tab
The NetworkPolicy details  code is not console

**Pull request merged**